keps/sig-network/2433-topology-aware-hints/kep.yaml

```diff
@@ -0,0 +1,55 @@
+title: Topology Aware Hints
+kep-number: 2433
+authors:
+  - "@robscott"
+owning-sig: sig-network
+status: implementable
+creation-date: 2021-02-04
+reviewers:
+  - "@andrewsykim"
+  - "@bowei"
+  - "@dcbw"
+  - "@thockin"
+approvers:
+  - "@thockin"
+  - "@wojtek-t"
+prr-approvers:
+  - "@wojtek-t"
+
+see-also:
+  - "https://docs.google.com/document/d/1ZzUoFY1SrdjVefl7gVOJZJLt1I1LHttw8pcX95nlgMY/edit?usp=sharing"
+  - "github.com/kubernetes/enhancements/blob/master/keps/sig-network/2004-topology-aware-subsetting"
+  - "github.com/kubernetes/enhancements/blob/master/keps/sig-network/2030-topology-aware-proxying"
+  - "github.com/kubernetes/enhancements/blob/master/keps/sig-network/2086-service-internal-traffic-policy"
+replaces:
+  - "github.com/kubernetes/enhancements/tree/master/keps/sig-network/536-topology-aware-routing"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+# latest-milestone: "v1.21"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.21"
+  beta: "v1.22"
+  stable: "v1.24"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: TopologyAwareHints
+    components:
+      - kube-controller-manager
+      - kube-proxy
+disable-supported: true
+
+# The following PRR answers are required at beta release
+metrics:
+  - endpoint_slice_controller/endpoint_overload_by_zone
```

Comment:
remove - on longer exists in the description

Reply:
Updated, thanks!